tests/serialize/structures.n3

```diff
@@ -0,0 +1,35 @@
+
+
+:MLK :name "Martin Luther King, Jr";
+  :born 1929-01-17; :died 1968-01-15;
+  :spouse [
+    :name "Coretta Scott King";
+    is :spouse of [
+      a :Marriage;
+      :spouse :MLK;
+      :offspring ( :MLK3 :DSK :YK [:name "Bernice King"]);
+    ]
+  ];
+
+  :wikipediapage <https://en.wikipedia.org/wiki/Martin_Luther_King_Jr.>.
+
+:MLK3 :name "Martin Luther King III".
+:DSK :name "Dexter Scott King".
```

Comment:
If we're now sorting by s, then p, then o, then shouldn't this be the first statement in the file?